CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.9.0
```

Comment:
Not sure which PR will be merged first #109 or this one. So, I write it like that.

Reply:
You are going to need to rebase the PR